README.md

```diff
@@ -415,3 +416,17 @@ android {
 * The library logs an error message about a not found class: Rejecting re-init on previously-failed class java.lang.Class<android.support.v4.app.JobIntentService$JobServiceEngineImpl>
 
   The message above stems from certain versions of the Android support libraries. JobServiceEngineImpl is part of Android Oreo (8, SDK 26) and newer only. The support library catering for supporting services on earlier versions of Android runs internal checks for which implementation is available to it. As part of the checks, it outputs the message as an informational feedback only.
+
+* Timber Lint warnings get exposed to my app even though I don't (want to) use Timber.
+
+  The solution for the time being is to disable linting for those specific lint warning in your app:
```

Comment:
Can we say for production? because it will helpful for the developer locally but not in production towards the app users

Reply:
Just to clarify - the users of an app will never see this.

What happens is:

- `Raygun4Android:provider` uses Timber (a library for better logging on the device's logcat)
- Timber comes with its own set of Android (Studio) Lint rules and they're opinionated.
- Due to how Android's toolchain works, the linting rules are passed "downstream" to any modulde/app that uses a library (our `:provider`) that uses Timber. In this case to `Raygun4Android:app`.
- So, anything that *uses* our `:provider` gets the Timber linting rules passed through to it, regardless if that app is using Timber itself or not (many apps do anyway).
- Effectively that means that an app developer not using Timber in their app sees linting rules and might be confused. From here the options for them are to switch off linting in their IDE completely or ignore the lint rules in their code etc - but the cleanest solution is currently to add this disable statement into the build.

So, this is not inherently a `debug` or `release` issue. In their app, they should do this anyway if they don't use Timber themselves.